### DISCUSSION TITLE
Is this `important!` necessary?

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#1069] [Q&A]
Hi @ppoffice 

I wonder if this [`important!`](https://github.com/ppoffice/hexo-theme-icarus/blob/30efe838c5eaa8885da1e33e986f9b1ea1866802/include/style/article.styl#L116) is necessary? Thanks.

**Comment by ppoffice** [comment]
Yeah, probably not.
